setup.cfg

```diff
@@ -36,9 +36,6 @@ setup_requires =
 install_requires =
     numpy
 
-[options.extras_require]
-cmawm = scipy
-
```

Comment:
Perhaps keeping scipy dependency optional and using it if available might be safer as a first step. What do you think?

```python
try:
    from scipy.stats import chi2 as chi2_ppf
    from scipy.stats import norm as norm_cdf
except ImportError:
    from cmaes._stats import chi2_ppf
    from cmaes._stats import norm_cdf
```

Reply:
sounds good. will update.

Reply:
Thank you!
By the way, we plan to release the new version at 4pm JST today. If you are busy today, I can work on it in the following pull request later :+1:

Reply:
sorry I don't think I can do this today.

Reply:
No problem! Thank you for letting me know.

I will work on it later 👍